util/project_allocation_mapper.py

```diff
@@ -113,20 +113,21 @@ def _create_ticket_for_pi_request(self, user):
                                  requestor = "us@tacc.utexas.edu")
         rt.createTicket(ticket)
 
+    '''
+    get all projects
+    returns list of tas_proj_obj, sorted by most recent allocation
+    allocations in each object are sorted newest to oldest
+    
+    '''
     def get_all_projects(self):
```

Comment:
We've been slowly aligning on using [Google's docstring style](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) for code documentation like this. In that model, this would look a bit like this (note placement directly under `def`):

```suggestion
    def get_all_projects(self):
        """Get all projects and their allocations for all users.
        
        Returns:
            List[pytas.Project]: a list of project TAS representations,
                sorted by most recent allocation. Allocations in each
                object are sorted newest to oldest.
```